src/constants/data.ts

```diff
@@ -62,6 +62,7 @@ export interface Progress {
 export interface NextExercise {
   disciplineId: number
   exerciseKey: number
+  disciplineTitle: string
```

Comment:
We do not want to save the title of a discipline, as it can change in the cms and wouldn't be updated in the app.

Maybe you should base this PR on LUN-339, the HomeScreen components are refactored there and there is also a request made to get the disciplineTitle, so it does not need to be saved locally.

Reply:
removed